src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -135,6 +135,8 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Split_string_literals_on_enter}" />
                     <CheckBox x:Name="Report_invalid_placeholders_in_string_dot_format_calls"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Report_invalid_placeholders_in_string_dot_format_calls}" />
+                    <CheckBox x:Name="Underline_reassigned_variables"
```

Comment:
I'd like an option similar to inline-hint that I can invoke the tagger on demand. I can definitely see myself using this feature but don't want all my locals underlined all the times.

Reply:
> but don't want all my locals underlined all the times.

then make less mutable variables :D

the idea is def interesting though!